config/faker-generator.js

```diff
@@ -0,0 +1,18 @@
+const _ = require('underscore');
+const fs = require('fs');
+const files = fs.readdirSync('./test/fixtures/config');
+
+_.each(files, (file) => {
+  const entity_factory = require(`../test/fixtures/config/${file}`);
+  const dest = `./test/fixtures/collections/${file.split('.')[0]}.json`;
+
+  const collection = _.times(1000, () => {
+    return entity_factory.generate();
+  });
+
+  fs.writeFile(dest, JSON.stringify(collection), (e) => {
```

Comment:
might want to add `JSON.stringify(collection, null, 2);` here.  It'll make the files readable

Reply:
was wondering if there was some magic that could do that, but I only looked at the JSON object, not down into the `stringify` method itself 🙃